tests/integration/conftest.py

```diff
@@ -246,6 +282,66 @@ def single_passing_test_step_for_mk8s(_zigzag_runner_factory, mk8s_config_file,
     return zz_runner
 
 
+@pytest.fixture(scope='session')
+def custom_build_url_config_file(tmpdir_factory):
+    """A config for zigzag used by the ASC team
+
+    Returns:
+        str : a path to a config file
+    """
+
+    filename = tmpdir_factory.mktemp('data').join('config_file.json').strpath
+    config_json = \
+        """{
+              "pytest_zigzag_env_vars": {
+              },
+              "zigzag": {
+                "test_cycle": "{{ ZZ_INTEGRATION_TEST_CYCLE }}",
+                "project_id": "{{ ZZ_INTEGRATION_PROJECT_ID }}",
+                "module_hierarchy": [
+                  "node1",
+                  "node2"
+                ],
+                "build_url": "https://github.com/rcbops/zigzag",
+                "build_number": "4"
+              }
+            }"""
+
+    with open(filename, 'w') as f:
+        f.write(config_json)
+
+    return filename
+
+
+@pytest.fixture(scope='session')
+def custom_hierarchy_config_file(tmpdir_factory):
+    """A config for zigzag used by the ASC team
```

Comment:
Description is vague.

Reply:
addressing in fb1987cbd8f4107ef7aba96c807d6cadfa7074dd

Reply:
Not resolved.